apps/televisit-demo/backend/output.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
Remove this file or replace the output values with placeholders?

Reply:
I looked at the `deploy.sh` script and it does not require this file. It contains values for your environment and should not be checked in.